notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb

```diff
@@ -0,0 +1,588 @@
+    "            else:\n",
+    "                # encode numpy array to jpg\n",
+    "                imgbox.value = cv2.imencode(\n",
+    "                    \".jpg\",\n",
+    "                    frame,\n",
+    "                    params=[cv2.IMWRITE_JPEG_QUALITY, 90],\n",
+    "                )[1].tobytes()\n",
+    "\n",
+    "            engine3D.renderer.render(engine3D.scene, engine3D.cam)\n",
+    "\n",
+    "    except KeyboardInterrupt:\n",
+    "        print(\"Interrupted\")\n",
+    "    except RuntimeError as e:\n",
+    "        print(e)\n",
+    "    finally:\n",
+    "        clear_output()\n",
+    "        if player is not None:\n",
+    "            # stop capturing\n",
+    "            player.stop()\n",
+    "        if use_popup:\n",
+    "            cv2.destroyAllWindows()\n",
+    "        if skeleton_set:\n",
+    "            engine3D.scene_remove(skeleton_set)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "344840a6-9660-4a11-8b05-729ac2969e28",
+   "metadata": {},
+   "source": [
+    "## Run\n",
+    "\n",
+    "### Run Live Pose Estimation\n",
+    "\n",
+    "Run using a webcam as the video input. By default, the primary webcam is set with `source=0`. If you have multiple webcams, each one will be assigned a consecutive number starting at 0. Set `flip=True` when using a front-facing camera. Some web browsers, especially Mozilla Firefox, may cause flickering. If you experience flickering, set `use_popup=True`.\n",
+    "\n",
+    "*Note:*\n",
+    "\n",
+    "*1. To use this notebook with a webcam, you need to run the notebook on a computer with a webcam. If you run the notebook on a server (e.g. Binder), the webcam will not work.*\n",
+    "         \n",
+    "*2. Popup mode may not work if you run this notebook on a remote computer (e.g. Binder).*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2d1a143-afcb-4f22-a4cc-657a080b70bf",
+   "metadata": {},
+   "source": [
+    "Use the following method, you can click and move your mouse over the picture on the left to interact."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f82e298-5912-48c7-90b5-339aea3c177d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "run_pose_estimation(source=0, flip=True, use_popup=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ee7b3c4-a143-4d54-bf5d-0aa758f99928",
+   "metadata": {},
+   "source": [
+    "### Run Pose Estimation on a Video File\n",
+    "\n",
+    "If you don't have a webcam, you can still run this demo with a video file. Any [format supported by OpenCV](https://docs.opencv.org/4.5.1/dd/d43/tutorial_py_video_display.html) will work. \n",
+    "\n",
+    "\n",
+    "You can click and move your mouse over the picture on the left to interact."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "da463932-dee0-4f0f-8379-133d16fce3a6",
+   "metadata": {},
```

Comment:
Please just commit as it is. It will change the behavior only for tests.
```suggestion
   "metadata": {
    "test_replace": {
     "skip_frames=10": "skip_frames=675"
    }
   },
```